compose.cc.yml

```diff
@@ -0,0 +1,133 @@
+services:
```

Comment:
I think we could accomplish the same without having to copy most of the `compose.yml` file here. I'm happy to just add the `cc` service to the `compose.dev.yml` file by now, so that it will start whith the command `start --up --dev`. In this case we can remove the `cp` step in the main readme.

Alternatively, we can have this compose.cc.yml file only with the `cc` service so that we can start it all with:

`docker compose -f compose.yml -f compose.dev.yml -f compose.cc.yml up`

And that could be eased with a `--cc`parameter in the `start.sh` script. But I don't think we really need it.

Reply:
OK I'll add it into compose.dev.yml, thank you!